wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -234,6 +244,9 @@ public void load(Device device) throws ModelException, IOException {
             }
 
             ZooModel<I, O> m = builder.build().loadModel();
+            if (engineName == null) {
+                engineName = m.getNDManager().getEngine().getEngineName();
```

Comment:
check if this collide with the `HF_MODEL_ID` design in LLM inference for infer engine